src/WebUI/WorkLoads/DeploymentsTable.tsx

```diff
@@ -141,11 +157,18 @@ export class DeploymentsTable extends BaseComponent<IDeploymentsTableProperties,
         deployment: V1Deployment,
         replica: V1ReplicaSet,
         index: number,
-        replicaSetLength: number): IDeploymentReplicaSetItem {
+        replicaSetLength: number,
+        imageService: IImageService | undefined): IDeploymentReplicaSetItem {
         // todo :: annotations are taken from deployment for the first replica, rest of the replicas from respective replicas
         const annotations: {
             [key: string]: string;
         } = index === 0 ? deployment.metadata.annotations : replica.metadata.annotations;
+        const podslist = StoreManager.GetStore<PodsStore>(PodsStore).getState().podsList;
+        const pods: V1Pod[] = podslist && podslist.items || [];
+        const imageId = Utils.getImageId(Utils.getFirstImageName(replica.spec.template.spec), replica.spec.template.metadata, pods);
+        if (DeploymentsTable._imageList.length <= 0 || DeploymentsTable._imageList.findIndex(img => equals(img, imageId)) < 0) {
```

Comment:
equals ==> if this has ignore case option use it.

Reply:
done